package.json

```diff
@@ -4,16 +4,19 @@
   "private": true,
   "author": "Matthew Brandly",
   "license": "MIT",
+  "main": "src/angular-youtube-embed.js",
   "repository": {
     "type": "git",
     "url": "git://github.com/brandly/angular-youtube-embed.git"
   },
   "homepage": "http://brandly.github.io/angular-youtube-embed/",
   "scripts": {
-    "postinstall": "bower install",
     "start": "gulp",
     "test": "gulp test"
   },
+  "dependencies": {
+    "angular": ""
```

Comment:
would it be better to list `angular` under [`peerDependencies`](https://docs.npmjs.com/files/package.json#peerdependencies)? honestly i've never used them, but it seems more appropriate